### ISSUE TITLE
`NoClassDefFoundError` running v1.0.4 in batch mode

### ISSUE THREAD

**Issue by ararslan** [OHNLP/MedXN#11]
I downloaded and extracted the zip file for the most recent tag but attempting to run it in batch mode on Linux per the example in the README produces an error:
```
$ java -cp resources:MedXN.jar org.ohnlp.medxn.Main /tmp/input-files/ out.txt
Exception in thread "main" java.lang.NoClassDefFoundError: org/ohnlp/medtagger/cr/FileSystemReader
        at org.ohnlp.medxn.Main.main(Main.java:23)
Caused by: java.lang.ClassNotFoundException: org.ohnlp.medtagger.cr.FileSystemReader
        at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:581)
        at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:178)
        at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:522)
        ... 1 more
```
Was the `cr` there meant to be `crs`?

**Comment by qqndrew** [comment]
This particular class should be from a different dependency -- from OHNLP MedTagger (https://github.com/OHNLP/MedTagger/blob/master/src/main/java/org/ohnlp/medtagger/cr/FileSystemReader.java). The NoClassDefFound might be a packaging issue from when we mavenized/moved to github actions/improper shading.

~~I'll look into fixing this directly in the MedXN release, in the meantime I would suggest just downloading the latest MedTagger release from https://github.com/OHNLP/MedTagger and adding to your classpath~~
Edit: Please try MedXN Release v1.0.5, which should have this fixed

Thanks for bringing this to our attention

**Comment by qqndrew** [comment]
Issue caused by improper shaded artifact being included by the ant build script for release 
https://github.com/OHNLP/MedXN/blob/master/pom.xml#L85
causes shaded artifact to have different name from what is expected/included in the final release zip.
This should be fixed as of MedXN release v1.0.5, please give that a try.

**Comment by ararslan** [comment]
Looks like v1.0.5 works great! Thanks!
